### ISSUE TITLE
get_system_wine_version raises subprocess.CalledProcessError — Lutris hangs

### ISSUE THREAD

**Issue by MKuranowski** [lutris/lutris#1453]
This issue is very similar to #1404, the error is just in a different place. `wine --version` command sometimes fails, and in this case subprocess.CalledProcessError is raised, not OSError.
Such error completely hangs Lutris, and requires a restart.

Possible fix is the same as in #1404, but in file [lutris/util/wine.py](https://github.com/lutris/lutris/blob/master/lutris/util/wine.py), line 183.

-----

Traceback:
```
subprocess.CalledProcessError: Command '['/opt/wine-staging/bin/wine', '--version']' returned non-zero exit status 127.
/opt/wine-staging/bin/wine: relocation error: /opt/wine-staging/bin/wine: symbol wine_get_patches version WINE_1.0 not defined in file libwine.so.1 with link time reference
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/lutris/gui/gameviews.py", line 327, in popup_contextual_menu
    self.contextual_menu.popup(event, game_row)
  File "/usr/lib/python3/dist-packages/lutris/gui/gameviews.py", line 522, in popup
    game = game or Game(game_id)
  File "/usr/lib/python3/dist-packages/lutris/game.py", line 56, in __init__
    self.load_config()
  File "/usr/lib/python3/dist-packages/lutris/game.py", line 106, in load_config
    game_config_id=self.game_config_id)
  File "/usr/lib/python3/dist-packages/lutris/config.py", line 168, in __init__
    self.update_cascaded_config()
  File "/usr/lib/python3/dist-packages/lutris/config.py", line 198, in update_cascaded_config
    self.system_config.update(self.get_defaults('system'))
  File "/usr/lib/python3/dist-packages/lutris/config.py", line 269, in get_defaults
    options_dict = self.options_as_dict(options_type)
  File "/usr/lib/python3/dist-packages/lutris/config.py", line 280, in options_as_dict
    if self.runner_slug
  File "/usr/lib/python3/dist-packages/lutris/sysoptions.py", line 353, in with_runner_overrides
    runner = runner()
  File "/usr/lib/python3/dist-packages/lutris/runners/winesteam.py", line 111, in __init__
    super(winesteam, self).__init__(config)
  File "/usr/lib/python3/dist-packages/lutris/runners/wine.py", line 193, in __init__
    'default': get_default_version(),
  File "/usr/lib/python3/dist-packages/lutris/util/wine.py", line 166, in get_default_version
    installed_versions = get_wine_versions()
  File "/usr/lib/python3/dist-packages/lutris/util/wine.py", line 124, in get_wine_versions
    version = get_system_wine_version(WINE_PATHS[build])
  File "/usr/lib/python3/dist-packages/lutris/util/wine.py", line 182, in get_system_wine_version
    version = subprocess.check_output([wine_path, "--version"]).decode().strip()
  File "/usr/lib/python3.6/subprocess.py", line 336, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.6/subprocess.py", line 418, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/opt/wine-staging/bin/wine', '--version']' returned non-zero exit status 127.
```

**Comment by strycore** [comment]
need reproduction steps

**Comment by Medath** [comment]
It seems to me like it's already fixed by 81fd71eefc4400428ac215aa4897e8a167eea70c.
Line 183 on `master` is the same line that was changed by 81fd71eefc4400428ac215aa4897e8a167eea70c on `next`

**Comment by MKuranowski** [comment]
@Medath oh right, if `next` is going to be merged with `master`, then #81fd71e fixed this issue
